runtimes/go/storage/objects/options.go

```diff
@@ -33,10 +33,25 @@ func (o withVersionOption) attrsOption() {}
 //publicapigen:keep
 func (o withVersionOption) existsOption() {}
 
+//publicapigen:keep
+func (o withTtlOption) uploadUrlOption() {}
+
 func (o withVersionOption) applyDownload(opts *downloadOptions) { opts.version = o.version }
 func (o withVersionOption) applyRemove(opts *removeOptions)     { opts.version = o.version }
 func (o withVersionOption) applyAttrs(opts *attrsOptions)       { opts.version = o.version }
 func (o withVersionOption) applyExists(opts *existsOptions)     { opts.version = o.version }
+func (o withTtlOption) applyUploadUrl(opts *uploadUrlOptions)   { opts.ttl = o.ttl }
+
+// WithTtl is used for signed URLs, to specify the lifetime of the generated
+// URL, in seconds. The max value is seven days.
+func WithTtl(ttl uint64) withTtlOption {
```

Comment:
Go convention is `TTL` not `Ttl`. Also make it take a `time.Duration` instead of an `uint64`
```suggestion
func WithTTL(ttl time.Duration) withTTLOption {
```

Reply:
Done